services/catarse/spec/queries/billing/payments/can_be_expired_query_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Billing::Payments::CanBeExpiredQuery, type: :query do
+  subject(:query) { described_class.new }
+
+  describe '#call' do
+    let!(:payment_with_pix) { create(:billing_payment, :with_pix, :waiting_payment) }
+    let!(:payment_with_boleto) { create(:billing_payment, :with_boleto, :waiting_payment) }
+
+    before do
+      payment_with_boleto.boleto.update!(expires_on: '2021-01-01')
+      payment_with_pix.pix.update!(expires_on: '2021-01-01')
+
+      Billing::PaymentStateMachine.states.each do |state|
+        create(:billing_payment, :with_boleto, state: state)
+        create(:billing_payment, :with_pix, state: state)
+      end
+    end
+
+    it 'returns expired payments' do
```

Comment:
**Sugestão**
Melhorar a descrição desse `it` pois essa mensagem leva a entender que ele vai retornar os pagamentos expirados, mas na verdade ele retorna os pagamentos que podem ser expirados.